lib/helpers.js

```diff
@@ -10,6 +10,11 @@ const PRETTIER_CORE_OPTIONS = [
   'bracketSpacing',
 ];
 
+const EMBEDDED_SCOPE = [
+  'text.html.vue',
+  'text.html.basic',
+];
```

Comment:
we could make them an option, but I'm not sure how used it will be? We can do that in a new PR if we want that.